assets/stylesheets/bootstrap/_type.scss

```diff
@@ -3,6 +3,19 @@
 // --------------------------------------------------
 
 
+@function em($pixels, $context: $browser-context) {
```

Comment:
And by explanation, I mean things like why ems are used and where pixels come into it.